cohort-engine/src/main/java/com/ibm/cohort/engine/parameter/DatetimeParameter.java

```diff
@@ -22,6 +22,6 @@ public DatetimeParameter(String value) {
 	
 	@Override
 	public Object toCqlType() {
-		return new DateTime( getValue().replace("@",""), OffsetDateTime.now().getOffset() );
+		return new DateTime(getValue().replace("@",""), ZoneOffset.UTC);
```

Comment:
Does this mean that all input datetimes will always be interpreted as UTC regardless of where the engine is running?  I think we went over this in one of our standup sessions, but it's all a blur at this point.

Reply:
The second argument is used for the timezone if no timezone exists in the String value used to create the DateTime.

So "2020-01-03T11:30:00+05:00" will be GMT+5, but "2020-01-03" will be UTC.